src/_locales/en_US/messages.json

```diff
@@ -131,8 +131,12 @@
         "message": "Import",
         "description": ""
     },
+    "options_hide_social_widgets": {
+        "message": "Don't replace the following widgets:",
+        "description": "Multiple selection box on the widget replacement tab"
+    },
     "options_social_widgets_checkbox": {
-        "message": "Replace social widgets",
+        "message": "Enable widget replacement",
         "description": "Checkbox label on the general settings page"
```

Comment:
It's now on the Widget Replacement tab on the options page.

Reply:
Ah, yes